src/ensembl/production/metadata/updater/core.py

```diff
@@ -528,6 +511,7 @@ def get_or_new_genebuild(self, species_id, meta_session, source=None, existing=F
         # The assembly accession and genebuild version are extracted from the metadata of the species
         assembly_accession = self.get_meta_single_meta_key(species_id, "assembly.accession")
         genebuild_version = self.get_meta_single_meta_key(species_id, "genebuild.version")
+        genebuild_start_date = self.get_meta_single_meta_key(species_id, "genebuild.start_date")
```

Comment:
Please move this down one line to keep the logic in order.

Reply:
I didn't but I added the test that at least those three values are mandatory (although I'll check them as mandatory for the DC to spot this as well before hand)